pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -206,14 +207,14 @@ func (ca *cloudCIDRAllocator) retryParams(nodeName string) (bool, time.Duration)
 }
 
 func nodeUpdateRetryTimeout(count int) time.Duration {
-	timeout := updateRetryTimeout
-	for i := 0; i < count && timeout < maxUpdateRetryTimeout; i++ {
+	timeout := updateRetryTimeoutMilliseconds
+	for i := 0; i < count && timeout < maxUpdateRetryTimeoutMilliseconds; i++ {
 		timeout *= 2
 	}
-	if timeout > maxUpdateRetryTimeout {
-		return maxUpdateRetryTimeout
+	if timeout > maxUpdateRetryTimeoutMilliseconds {
+		timeout = maxUpdateRetryTimeoutMilliseconds
 	}
-	return timeout
+	return time.Duration(rand.Int31n(timeout)) * time.Millisecond
```

Comment:
I think that it's not how we should do that. The expected value of that is 1/2 of what we have now. Also the variation here is potentially too big.

Instead of that, I think we should make it more like "uniformly distributed between [1/2*timeout, timeout].
Or timeout + some random number?
Or uniformly distributed between [0.75*timeout, 1.25*timeout] (I think it's what Jitter does..)

But maybe good to get @bowei `s opinion too.

Reply:
I changed it to `[0.5,1.5]*timeout` to keep the expectation same. The variance I think should be high so that we can actually spread out the calls enough.
WIll wait for Bowei.